File: Beginner/Salary_With_Bonus_1009/Main.java
import java.math.BigDecimal;
import java.util.Scanner;
import java.math.RoundingMode;


public class Main {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);
        String sellerName = sc.nextLine();
        double fixedSalary = sc.nextDouble();
        double totalSale = sc.nextDouble();
        double total = fixedSalary + (totalSale * 0.15);
        
        System.out.println(String.format("TOTAL = R$ %.2f" , total));
        
        sc.close();
    }
}

File: Beginner/Salary_With_Bonus_1009/main.go
package main

import (
	"fmt"
)

func main() {
	var sellerName string
	fmt.Scan(&sellerName)
	var fixedSalary float64
	fmt.Scan(&fixedSalary)
	var totalSale float64
	fmt.Scan(&totalSale)

	total := fixedSalary + (0.15 * totalSale)

	fmt.Printf("TOTAL = R$ %.2f\n", total)
}
